decidim-meetings/spec/system/live_meeting_access_spec.rb

```diff
@@ -20,16 +20,216 @@ def visit_meeting
   end
 
   context "when online meeting is live" do
-    let(:meeting) { create :meeting, :published, :online, :live, component: component }
+    shared_examples "iframe access levels" do |embedding_type|
+      context "when the iframe access level is for all visitors" do
+        before do
+          meeting.iframe_access_level_all!
+        end
 
-    it "shows the link to the live meeting streaming" do
-      visit_meeting
+        context "and user is signed in" do
+          before do
+            login_as user, scope: :user
+          end
+
+          it "shows the meeting link embedded" do
```

Comment:
I don't like the "meeting link embedded" as a concept, but then I realized it makes perfect sense with the "iframe_embed_type" of "open_in_new_tab" 🙈 🤷🏽